### DISCUSSION TITLE
[@types/marked] breaking change with marked@4.0.0

### DISCUSSION THREAD

**Discussion by Vogel612** [DefinitelyTyped/DefinitelyTyped#57046] [Issues with a @types package] [answered]
Types for `marked` are outdated since the release of version 4.0.0 6 days ago breaking the default export: https://github.com/markedjs/marked/releases/tag/v4.0.0. There also is some outstanding issue around missing a namespace that probably can be closed along the way.

[a comment by an automated account is not shown]

**Comment by UziTech** [comment, marked as answer]
Should be fixed in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/56967

**Reply by Vogel612** [reply]
Nice, so it's just waiting for review and publish to get around to it.

**Reply by UziTech** [reply]
Yup
